src/fidesops/service/connectors/saas_connector.py

```diff
@@ -98,14 +101,6 @@ def create_client(self) -> AuthenticatedClient:
         logger.info(f"Creating client to {uri}")
         return AuthenticatedClient(uri, self.client_config, self.secrets)
 
-    @staticmethod
-    def get_value_by_path(dictionary: Dict, path: str) -> Dict:
-        """Helper method to extract an arbitrary data path from a given dictionary"""
-        value = dictionary
-        for key in path.split("/"):
-            value = value[key]
-        return value
```

Comment:
Nice swap to use pydash instead of this custom method - pydash has some great features.